docker/pypi/wmagent-couchdb/README.md

```diff
@@ -0,0 +1,300 @@
+REPOSITORY                                        TAG       IMAGE ID       CREATED         SIZE
+local/couchdb                                     3.2.2     2ec9e59aa0e9   1 minute ago     819MB
+```
+* Using the wrapper script to build and upload CouchDB to registry.cern.ch:
+```
+./couchdb-docker-build.sh -t 3.2.2 -p
+```
+
+### Running a CouchDB container
+
+We can run from the local repository or from upstream CERN registry.
+
+* Running from a local build:
+
+```
+cmst1@vocms0290:wmagent-couchdb $ ./couchdb-docker-run.sh -t 3.2.2
+Starting the couchdb:3.2.2 docker container with the following parameters:  --user cmst1
+8decd12e153d74c9de48764b8d3faf975d7d52897c5b0fc1032e6fef7a7c74dd
+
+cmst1@vocms0290:wmagent-couchdb $ docker ps
+CONTAINER ID   IMAGE                             COMMAND           CREATED          STATUS          PORTS     NAMES
+8decd12e153d   local/couchdb:3.2.2               "./run.sh"        12 seconds ago   Up 11 seconds             couchdb
+```
+
+* Running from CERN registry:
+```
+cmst1@vocms0290:wmagent-couchdb $ ./couchdb-docker-run.sh -t 3.2.2 -p
+Pulling Docker image: registry.cern.ch/cmsweb/cdb:3.2.2
+3.2.2: Pulling from cmsweb/couchdb
+Digest: sha256:61f798b55a1c743686e1568509975308dc07b5b24486894053d6a312983c4af6
+Status: Downloaded newer image for registry.cern.ch/cmsweb/couchdb:3.2.2
+registry.cern.ch/cmsweb/couchdb:3.2.2
+Starting the couchdb:3.2.2 docker container with the following parameters:  --user cmst1
+21d9c6598f35e627834d1b796460047605d6255cebc746d572289c7b418053ed
+
+cmst1@vocms0290:wmagent-couchdb $ docker ps
+CONTAINER ID   IMAGE                             COMMAND      CREATED         STATUS         PORTS     NAMES
+21d9c6598f35   registry.cern.ch/couchdb:3.2.2   "./run.sh"   7 seconds ago   Up 6 seconds             couchdb
+
+```
+
+* Killing the container directly from the host:
+```
+cmst1@vocms0290:wmagent-couchdb $ docker kill couchdb
+couchdb
+
+```
+
+* Connecting to a running container:
+```
+cmst1@vocms0290:wmagent-couchdb $ docker exec -it couchdb bash
+(CouchDB-3.2.2) [cmst1@vocms0290:data]$
+
+```
+
+* Fetching startup logs:
+```
+cmst1@vocms0265:wmagent-couchdb $ docker logs couchdb
+ME : data
+TOP : /
+ROOT : /
+CFGDIR : /data/srv/couchdb/3.2.2/config
+LOGDIR : /data/srv/couchdb/3.2.2/logs
+STATEDIR : /data/srv/couchdb/3.2.2/state
+KEYFILE : /data/srv/couchdb/auth//hmackey.ini
+
+COUCH_ROOT_DIR : /data
+COUCH_BASE_DIR : /data/srv/couchdb
+COUCH_STATE_DIR : /data/srv/couchdb/3.2.2/state
+COUCH_INSTALL_DIR : /data/srv/couchdb/3.2.2/install
+COUCH_CONFIG_DIR : /data/srv/couchdb/3.2.2/config
+<snip>
+Installing WorkQueue app into database: http://localhost:5984/workqueue
+Installing WorkQueue app into database: http://localhost:5984/workqueue_inbox
+start sleeping....zzz
+```
+
+### Managing the database service:
+
+All of the commands bellow must be run from inside the container.
+
+* General options:
+```
+(CouchDB-3.2.2) [cmst1@vocms0290:data]$ manage help
```

Comment:
Now that you added a new function called `print-settings`, please update this README accordingly.